main.rb

```diff
@@ -37,18 +20,4 @@ def select_option(app, num)
   end
 end
 
-def main
-  app = App.new
-  loop do
-    options
-    num = gets.chomp
-    break if num == '7'
-
-    select_option(app, num)
-    puts "\n"
-  end
-
-  puts 'Goodbye'
-end
-
 main
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you have got error with line `23`("main"), please fix it 🙏🏼 

![image](https://github.com/aima98/OOP-School-Library1/assets/63915024/7d067781-ed62-4427-bf92-9d806e750096)